### PR TITLE
Untap local formulas to fix "brew update" command

### PR DESCRIPTION
**Pull request by maxim-lobanov**
# Description
Related issues:
- https://github.com/actions/virtual-environments/issues/1869
- https://github.com/actions/virtual-environments/issues/1811

Untap local formulas to prevent issues on brew update

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by al-cheb — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

File: images/macos/provision/core/python.sh [hunk 1]
```diff
@@ -5,14 +5,15 @@ echo "Installing Python Tooling"
...
-/usr/local/bin/brew install python@3.8
-/usr/local/bin/brew install python@3.9
-/usr/local/bin/brew unlink python@3.9
-/usr/local/bin/brew unlink python@3.8
-/usr/local/bin/brew link python@3.8
+brew install python@3.8
+brew install python@3.9
+brew unlink python@3.9
+brew unlink python@3.8
+brew link python@3.8
```
Comment (AlenaSviridenko) [inline images/macos/provision/core/python.sh:12]: I thought we are removing this workaround here https://github.com/actions/virtual-environments/pull/1867/files, maybe makes sense to combine PRs?

Comment (maxim-lobanov) [reply images/macos/provision/core/python.sh:12]: I am not ready to merge that PR yet. I am waiting for Python tests to be ready.

**Review by AlenaSviridenko — APPROVED** [review]

File: images/macos/provision/core/openssl.sh [hunk 2]
```diff
@@ -2,18 +2,16 @@
...
 # Install OpenSSL 1.0.2t
 # https://www.openssl.org/policies/releasestrat.html - Version 1.0.2 will be supported until 2019-12-31 (LTS)
 # To preserve backward compatibility with ruby-toolcache
-/usr/local/bin/brew tap-new local/openssl
-FORMULA_PATH=$(/usr/local/bin/brew extract openssl local/openssl | grep "Homebrew/Library/Taps")
-/usr/local/bin/brew install $FORMULA_PATH
+brew tap-new local/openssl
+FORMULA_PATH=$(brew extract openssl local/openssl | grep "Homebrew/Library/Taps")
+brew install $FORMULA_PATH
+brew untap local/homebrew-openssl
```
Comment (MikeMcQuaid) [inline images/macos/provision/core/openssl.sh:14]:
```suggestion
brew tap-new --no-git local/openssl
FORMULA_PATH=$(brew extract openssl local/openssl | grep "Homebrew/Library/Taps")
brew install $FORMULA_PATH
```

File: images/macos/provision/core/python.sh [hunk 3]
```diff
@@ -5,14 +5,15 @@ echo "Installing Python Tooling"
...
 
 echo "Brew Installing Python 2"
 # Create local tap with formula due to python2 formula depreciation
-/usr/local/bin/brew tap-new local/python2
-FORMULA_PATH=$(/usr/local/bin/brew extract python@2 local/python2 | grep "Homebrew/Library/Taps")
-/usr/local/bin/brew install $FORMULA_PATH
+brew tap-new local/python2
+FORMULA_PATH=$(brew extract python@2 local/python2 | grep "Homebrew/Library/Taps")
+brew install $FORMULA_PATH
+brew untap local/homebrew-python2
```
Comment (MikeMcQuaid) [inline images/macos/provision/core/python.sh:19]:
```suggestion
brew tap-new --no-git local/python2
FORMULA_PATH=$(brew extract python@2 local/python2 | grep "Homebrew/Library/Taps")
brew install $FORMULA_PATH
```

**Pull request merged**